framework/decode/replay_options.h

```diff
@@ -41,25 +41,32 @@ struct ScreenshotRange
 
 struct ReplayOptions
 {
-    bool        enable_validation_layer{ false };
-    bool        sync_queue_submissions{ false };
-    bool        enable_debug_device_lost{ false };
-    bool        create_dummy_allocations{ false };
-    bool        omit_null_hardware_buffers{ false };
-    bool        quit_after_measurement_frame_range{ false };
-    bool        flush_measurement_frame_range{ false };
-    bool        flush_inside_measurement_range{ false };
-    bool        force_windowed{ false };
-    uint32_t    windowed_width{ 0 };
-    uint32_t    windowed_height{ 0 };
-    bool        force_windowed_origin{ false };
-    int32_t     window_topleft_x{ 0 };
-    int32_t     window_topleft_y{ 0 };
-    int32_t     override_gpu_index{ -1 };
-    std::string capture_filename;
-    bool        enable_print_block_info{ false };
-    int64_t     block_index_from{ -1 };
-    int64_t     block_index_to{ -1 };
+    bool                         enable_validation_layer{ false };
```

Comment:
If were going to change anything that isn't OXR specific, that should be in a separate commit before the OXR specific one, s.t. we can put it on dev.

Reply:
Now in a separate commit.